solr/core/src/java/org/apache/solr/core/NodeConfig.java

```diff
@@ -89,6 +89,8 @@
   private final boolean fromZookeeper;
   private final String defaultZkHost;
 
+  private final String configSetServiceClass;
```

Comment:
Move up to the other classes to be more organized

Reply:
as i see ,there is no ready-made class to store the property 'configSetServiceClass'. it seem that make a new class to store  'configSetServiceClass' and  responsible for creating  custome ConfigSetService instance. is that ok?

Reply:
Sorry; I wasn't clear.  I simply mean move the field higher to be amongst the other fields that end with "Class" :-)

Reply:
Ok, I got it